Tutorials/ASP.NET/Blazor.ServerSide/CS/Pages/Error.razor

```diff
@@ -0,0 +1,16 @@
+﻿@page "/error"
+
+
+<h1 class="text-danger">Error.</h1>
+<h2 class="text-danger">An error occurred while processing your request.</h2>
+
+<h3>Development Mode</h3>
+<p>
+    Swapping to <strong>Development</strong> environment will display more detailed information about the error that occurred.
+</p>
+<p>
+    <strong>The Development environment shouldn't be enabled for deployed applications.</strong>
+    It can result in displaying sensitive information from exceptions to end users.
```

Comment:
Is line 13 related to line 12? If so, can we combine them and say:

`The Development environment should not be enabled for deployed applications, because it may display sensitive exception information to end users.`